web/test/mock/withStatefulUserData.tsx

```diff
@@ -3,7 +3,7 @@ import { UserData } from "@/lib/types/types";
 import * as useUserModule from "@/lib/data-hooks/useUserData";
 import { getLastCalledWith } from "@/test/helpers";
 
-const updateSpy = jest.fn();
+export const updateSpy = jest.fn();
```

Comment:
this was not exported on purpose - the idea was that in any test where we need to check this, we should be using the statefulUserData and asserting against the currentUserData instead